tests/unit/test_preferences.py

```diff
@@ -47,22 +48,22 @@ def test_map_setting_valid_choice(self):
 
     def test_enum_setting_invalid_default_value(self):
         with self.assertRaises(ValidationException):
-            EnumStringSetting(3, choices=[0, 1, 2])
+            EnumStringSetting('3', choices=['0', '1', '2'])
```

Comment:
`EnumStringSetting` isn't supposed to take integers (its typed arguments enforce `str`...), and none of its usage uses integers - converted these all to strings to represent actual usage.